app/src/main/java/app/abhijit/iter/AttendanceActivity.java

```diff
@@ -70,38 +70,38 @@
 public class AttendanceActivity extends AppCompatActivity
         implements NavigationView.OnNavigationItemSelectedListener{
 
-    private Context mContext;
-    private SharedPreferences mSharedPreferences;
-    private Cache mCache;
-    private Student mNewStudent;
-    private Student mOldStudent;
-    private boolean mPrefExtendedStats;
-    private int mPrefMinimumAttendance;
-    private ArrayList<SubjectView> mSubjectViews;
-    private SubjectAdapter mSubjectAdapter;
+    private Context context;
+    private SharedPreferences sharedPreferences;
+    private Cache Cache;
```

Comment:
Variable names must be in lowerCamelCase. The first character must be in lowercase.